var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -79,7 +79,7 @@ class Trilinos(CMakePackage, CudaPackage):
             description="Use nvcc-wrapper for CUDA build")
     variant('cuda_rdc', default=False,
             description='turn on RDC for CUDA build')
-    variant('cxxstd', default='11', values=['11', '14', '17'], multi=False)
+    variant('cxxstd', default='14', values=['11', '14', '17'], multi=False)
```

Comment:
No, this pr should not touch trilinos. If you do need `cxxstd=14` of it (why??), then simply depend on `trilinos cxxstd=14`.

Reply:
Didn't mean to push this change.